File: Java/TwoKnights.java
import java.util.Scanner;
import java.io.BufferedWriter;
import java.io.OutputStreamWriter;
import java.io.IOException;
import java.lang.StringBuilder;
import static java.lang.Math.pow;

public class TwoKnights {
  public static void main(String[] args) {

    Scanner sc = new Scanner(System.in);
    BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
    StringBuilder str = new StringBuilder();
    int n = sc.nextInt();
    int a = 1;

    while (a != n + 1) {
      if (a == 1) {
        str.append(0).append("\n");

      } else if (a == 2) {
        str.append(6).append("\n");

      }
      else if (a == 3) {
        str.append(28).append("\n");

      }
      else {
        long a = (pow(n,2)*(pow(n,2) - 1) / 2)
      }
      a += 1;
    }
  }

}

File: python/Introductory Problems/Two_Knights.py
from math import pow

n = int(input())
a = 1
while(a!=n+1):
  if a == 1 :
    result = 0
  elif a == 2:
    result = 6
  elif a == 3:
    result = 28
  else:
  
    result =( pow(a,2) * (pow(a , 2) - 1) / 2) - ((10 + (a - 4) * 4) * (a - 2)) - (4 + (a - 4 ) *  2 )
  a += 1  
  print(int(result))